analyses/country_template/docs/template.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: '<img src="https://drive.google.com/uc?id=118y5T73-iSUZoAMtoJGddxq9QzD_GDKX" style="height:40px;float:left; margin:10px" /><img src="https://drive.google.com/uc?id=1fHQUzF3ZjaoHj9KQ33-94dK_X1hcmjzW" style="height:50px;float:left; margin:10px" />'
+pagetitle: 'Docs Template'
+output:
+  html_document:
+    df_print: paged
+    toc: yes
+    toc_float: yes
+    toc_depth: 4
+  pdf_document:
+    toc: yes
+    toc_depth: '4'
+editor_options:
+  chunk_output_type: inline
+---
+
+<br> 
+<br>
+<br>
+
+# Documentation Template
+Analysis last updated `r Sys.Date()`
+
+<br>
+
```

Comment:
Do you think it'd be helpful to add a section on global options (remove scientific notations, do not print output of chunks, etc.)?

Also, do we want to follow the practice of loading an analysis script (.R) or reproducing parts of a full analysis in an Rmd? What about functions: should they be defined in a separate script?